### PR TITLE
editing entity

### PR DESCRIPTION
**Pull request by paubenemerito**

### CODE REVIEW

File: app/controllers/entities_controller.rb [hunk 1]
```diff
@@ -23,6 +27,19 @@ def create
     end
   end
 
+  def update
+    respond_to do |format|
+    @entity = Entity.find(params[:id])
```
Comment (jediU) [inline app/controllers/entities_controller.rb]: Wrong indentation

**Review by jediU — CHANGES_REQUESTED** [review]

File: app/controllers/entities_controller.rb (hunk 1 above)
Comment (rusbal) [reply app/controllers/entities_controller.rb]: To eliminate duplication, move this code into `def set_entity`

`@entity = Entity.find(params[:id])`

Add before action on top.

**Review by rusbal — CHANGES_REQUESTED** [review]
Remove duplication by creating a method that runs before specific actions.

File: app/views/entities/index.html.erb [hunk 2]
```diff
@@ -20,6 +20,7 @@
         <td><div class="money"><%= ent.transaction_fee %></div></td>
         <td><%= ent.description %></td>
         <td><%= link_to "Add Child", new_entity_path(parent_id: ent.id) %></td>
+        <td><%= link_to "Edit", edit_entity_path(ent.id) %></td>
```
Comment (rusbal) [inline app/views/entities/index.html.erb]: You may simplify

`link_to "Edit", edit_entity_path(ent)`

Comment (rusbal) [reply app/views/entities/index.html.erb]: For readability, change `ent` to `entity`.

File: app/controllers/entities_controller.rb [hunk 3]
```diff
@@ -59,8 +74,12 @@ def set_entity(create)
               end
   end
 
+  def find_params
```
Comment (rusbal) [inline app/controllers/entities_controller.rb]: Change `find_params` to `set_entity`

**Review by rusbal — APPROVED** [review]
🚢 :shipit:

**Pull request merged**